gemstone/tests/ios/GemTest/GemTestTests/GemTestTests.swift

```diff
@@ -10,27 +10,42 @@ import Gemstone
 @testable import GemTest
 
 extension Data {
+
+    init?(hex: String) {
```

Comment:
can you move this to extension? to avoid having inside `GemTestTests.swift`

Reply:
we can include normal swift code to Gemstone.xcframework later, currently only codegen code